### DISCUSSION TITLE
My student authentication application has been approved, and it has been a long time since it was approved, but the benefits have not been issued yet

### DISCUSSION THREAD

**Discussion by Luzkk** [community/community#139451] [GitHub Education]
### Hi everyone,

please help me

**Comment by samus-aran** [comment]
Hi @Luzkk  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
